config/routes.rb

```diff
@@ -4,6 +4,7 @@
   root to: redirect("/step-by-step-pages", status: 302)
 
   resources :step_by_step_pages, path: "step-by-step-pages" do
+    get "approve-2i-review", to: "review#approve_2i_review"
     post "approve-2i-review", to: "review#approve_2i_review"
```

Comment:
Have discussed IRL. Think it's best to encapsulate all the 2i review methods in one controller, rather than several small controllers - but happy to rename the methods.